src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -448,6 +451,51 @@ public void userProfileCreateUserWireMock(HttpStatus status) {
         );
     }
 
+    public void userProfileCreateUserWireMockWithExtraRoles() {
+
+        String body = null;
+        body = "{"
```

Comment:
Can we please do it in single line

Reply:
We can, although it has been done like this elsewhere and is more similar to JSON format this way - what do you think?

Reply:
it's fine. We can keep it as is.